flask_simplelogin/__init__.py

```diff
@@ -284,7 +311,8 @@ def login(self):
         )
 
         if is_logged_in():
-            flash(self.messages['is_logged_in'], 'primary')
+            message = self.messages['is_logged_in']
+            message.flash()
```

Comment:
We could simply do `self.messages['is_logged_in'].flash()` (this could be the case of lines 104-15, 326-327, 332-333, 339-340).

I just think that the repetition of `message.flash()` in five different places places might be a _code smell_: there might me something we could improve in `Message` — maybe use the `__call__` magic to short it to `self.messages['is_logged_in']()` would help, but I'm nor sure which is more readable and expressive. What do you think?

Reply:
Okay. I can do `self.messages['is_logged_in'].flash()` instead. I'm afraid I'm not familiar with `__call__` though.

Reply:
The `__call__` defines what happen when an instance of a call is called. For example:

```python
class Message:
  def __init__(self, text):
      self.text = text
  
  def __call__(self):
    print(self.text)


message = Message("hello world")
message()
```

Will print `hello world` : )